### PR TITLE
Extend issue_search with updated and closed, and add pr_search

### PR DESCRIPTION
**Pull request by AndreKurait**
Extend issue_search with updated and closed, and add pr_search.

Testing (Note these links may not persist, this testing can be replicated by opening the html files locally):

https://andrekurait.github.io/peternied.github.io/redirect/pr_search.html?owner=opensearch-project&repo=OpenSearch&merged-since-days=75&is-open=false

https://andrekurait.github.io/peternied.github.io/redirect/issue_search.html?owner=opensearch-project&repo=OpenSearch&closed-since-days=75&is-open=false

### CODE REVIEW

File: redirect/pr_search.html [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+      }
+
+      // Get parameters from URL
+      var owner = getQueryParam('owner');
+      var repo = getQueryParam('repo');
+      var tag = getQueryParam('tag');
+      var createdDays = parseInt(getQueryParam('created-since-days'), 10);
+      var updatedDays = parseInt(getQueryParam('updated-since-days'), 10);
+      var mergedDays = parseInt(getQueryParam('merged-since-days'), 10);
+      var closedDays = parseInt(getQueryParam('closed-since-days'), 10);
```
Comment (peternied) [inline redirect/pr_search.html:47]: What is the difference between merged and closed days?

**Review by peternied — COMMENTED** [review]
Nice, but rather than have two separate scripts could we combine this into a single script due to the amount of reuse.

Alternatively what do you think about breaking some of the functionality into separate files to reduce the duplication.

---

Also with all the extra fields, seems like adding at least a very basic input -> output matches tests across the board would ensure nothing is broken.

File: redirect/pr_search.html (hunk 1 above)
Comment (AndreKurait) [reply redirect/pr_search.html:47]: PRs can be closed without merging

**Comment by peternied** [comment]
Thanks for the contribution, I'll make the improvements I was getting at myself after merging.

**Pull request merged**